### ISSUE TITLE
[Enhancement]: Same book, different narrators

### ISSUE THREAD

**Issue by racemus** [advplyr/audiobookshelf#2396]
### Describe the feature/enhancement

Hi

I have audiobooks that are the same, but voiced by different narrators. Sometimes it's fascinating to listen to different voiceovers. I kept them within different subfolders within the book folder, but now with ABS I have to create a book folder for each narrator.

Also, it's not so convenient to have multiple, but the same books within the library. I tried to use subseries for it, but I can't create real subseries with it.

So, I would suggest updating Directory Structure rules for it. For example, use subfolders with narrators names, as I used before. And then give user choice which narrator to play somehow on front-end.

What do you think?

**Comment by advplyr** [comment]
Yeah progress would be an issue here unless we tracked them separately and selecting a different narrator would show the progress for that narrator. On the book covers we would have to pick one to show the progress of.

I can see this being useful but it's unlikely it gets added since it is a lot of work and only a small number of users would benefit from it.

**Comment by racemus** [comment]
> but it's unlikely it gets added since it is a lot of work and only a small number of users would benefit from it
Pure prioritization, I understand. It's a pity that I'm not a JS developer. Hope it will be made someday.

**Comment by DDriggs00** [comment]
Even if the full life-switching feature was not added, it would be cool to have support for multiple versions of the same book. E.g. the Wheel of Time with the new Rosemund Pike narrated versions, or the Lord of The Rings with its multiple narrators.

**Comment by manalive** [comment]
I also like this general idea, but it's kind of a problem to treat it as the same book because
- The narrator is very rarely the _only_ difference (usually, there are edition differences, translations, abridgments, additional forwards/prefaces, etc)
- Many times, the same "book" may actually have different chapter splits from different publishers
- as mentioned by others, matching progress is a challenge
- having a matching ebook version or supplemental/companion PDF for each narration adds UI complexity

What would be nice is being able to:
1. Identify distinct editions/adaptations/abridgments/translations/narrations of the same original 'work' and link them together
2. Collapse them or expand them in the list view just like Series
3. See the linked entries while on the detailed view
4. Star/Favorite or otherwise mark preferred versions of a work (or optimally give them a numeric order of preference)

When it comes to classics / great works, I often try to collect as many versions as possible and listen to them all over time. Example: Fellowship of the Ring
- English Narrator: Andy Serkis 
- English Narrator: Rob Ingles
- English Language Radio Drama: NPR 
- English Language Radio Drama: BBC Radio 4

**Comment by ZLoth** [comment]
As an example of books with different narrators:
* James Bond Books (Casino Royale is narriated first by Simon Vance, then the Celebrity Performance by Dan Stevens)
* The Lock-In series which I have organized as follows:
![image](https://github.com/advplyr/audiobookshelf/assets/6700159/db80eae5-9e67-4251-ab3f-71fe5986b9a9)
* Lord of the Rings

PLUS, there is the non-booktrack and booktrack versions.

**Comment by gravethief13** [comment]
I wasn't sure if this suggestion should have it's thread or not, but it seems somewhat related to what OP was talking about. My main issue is that when I have two or more versions of a book with different narrators and I want to listen to a specific one, I have to guess between them until I find it, or put the narrator's name in the cover art. Why not, for now at least, have a little sub line under the authors name when two books with the same title and author are displayed? Whether that be in the library or series view. Something like this:

<img width="1849" height="536" alt="Image" src="https://github.com/user-attachments/assets/6c7a3374-eef9-4fa4-96a0-71a7749faaec" />

If there's multiple narrators for a book, you could list a few then add in a little ... instead of 30 or so narrators showing up in that space.

**Comment by gravethief13** [comment]
Another thought, could also add in a way to differentiate "sets" of series with different narrators with brackets, assuming the same books with different narrators are eventually linked together in the future in a way in one of the above posts.



```
Terry Pratchett
	Discworld [Isis Audio]
		Book 01 - The Colour of Magic {Nigel Planer}
		Book 02 - The Light Fantastic {Nigel Planer}
		Book 03 - Equal Rites {Celia Imrie}
	Discworld [Penguin Audio]
		Book 01 - The Colour of Magic {Colin Morgan, Bill Nighy, Peter Serafinowicz}
		Book 02 - The Light Fantastic {Colin Morgan, Bill Nighy, Peter Serafinowicz}
		Book 03 - Equal Rites {Indira Varma, Bill Nighy, Peter Serafinowicz}
	Discworld [Read by Tony Robinson]
		Book 01 - The Colour of Magic {Tony Robinson}
		Book 03 - Equal Rites {Tony Robinson}
```

This might even work with the to-be-implemented-sometime subseries in subfolders structure. You could have the entire over-series have brackets if you have multiple versions of the whole over-series, or you could have the over-series have no brackets, and put brackets on different versions of the sub-series. 

This shouldn't effect the current uses of brackets because as far as I can tell, those only apply on the individual book level.

UI wise, it could be as simple as this:

<img width="1784" height="733" alt="Image" src="https://github.com/user-attachments/assets/900b7412-29ba-4ad1-8443-b6c71b208b49" />
